drivers/adc/adc_stm32.c

```diff
@@ -613,6 +614,12 @@ static int adc_stm32_calibrate(const struct device *dev)
 	}
 #endif /* CONFIG_SOC_SERIES_STM32H7X */
 
+	/* enable device runtime power management */
```

Comment:
I am not sure it is the best place to do that, you probably rather want to do that in `adc_stm32_init` instead